src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -103,6 +104,22 @@ public String getUrlForWiki(String wikiName, boolean addWWW) {
     }
   }
 
+  public  String getWikiGlobalURL(){
+    EnvType env = Configuration.getEnvType(this.env);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "env" which hides the field declared at line 14. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)